mods/ra/maps/exodus/exodus.lua

```diff
@@ -184,13 +184,6 @@ SendSovietParadrop = function()
 	Trigger.AfterDelay(sovietParadropTicks, SendSovietParadrop)
 end
 
-IdleHunt = function(unit)
-	Trigger.OnIdle(unit, unit.Hunt)
-	Trigger.OnCapture(unit, function()
-		Trigger.ClearAll(unit)
```

Comment:
Are we sure this OnCapture part isn't needed here?

Reply:
Looks like it's there to work around players using thieves by stealing Soviet tech. I don't believe they add much to the theme of the mission so I added a commit to remove thieves from this mission.